app/MMB/src/scenes/appleWallet/WalletContent.js

```diff
@@ -2,11 +2,29 @@
 
 import * as React from 'react';
 import { View } from 'react-native';
-import { StyleSheet, Color, Text } from '@kiwicom/mobile-shared';
-import { Translation } from '@kiwicom/mobile-localization';
+import {
+  StyleSheet,
+  Color,
+  Text,
+  AppleWalletButton,
+  PassBook,
+} from '@kiwicom/mobile-shared';
+import { Translation, Alert } from '@kiwicom/mobile-localization';
 
 import Passenger from './Passenger';
 import DateAndTime from './DateAndTime';
+import WalletContext from '../../context/WalletContext';
+
+const createAddHandler = pkpassUrl => async () => {
+  try {
+    await PassBook.addPass(pkpassUrl);
+  } catch (err) {
+    return Alert.translatedAlert(undefined, {
+      passThrough:
+        'There was an error during communication with our server. Please try again later',
```

Comment:
You need to create a translation for this in `ManageMyBooking.js`

Reply:
🤦‍♂️